python/tests/unit/arcticdb/version_store/test_update.py

```diff
@@ -450,7 +450,7 @@ def _create_product_candles_df(arr):
     assert_frame_equal(after_arctic, before_arctic)
 
 
-def test_update_non_sorted_exception(lmdb_version_store):
+def test_update_input_non_sorted_exception(lmdb_version_store):
```

Comment:
`not_sorted` not `non_sorted`

Reply:
there are plenty of examples in the tests where we use "non"...test_partial_write_non_contiguous, non_sorted_arr_1, for example